bool go_home
bool go_zero
geometry_msgs/Pose palm_goal_pose_world
---
bool success
trajectory_msgs/JointTrajectory plan_traj
